### ISSUE TITLE
No option to create graph for openupm version.

### ISSUE THREAD

**Issue by quabug** [alelievr/NodeGraphProcessor#126]
Unity 2020.2.3f1

1. Create a new project by UnityHub
2. Add `NodeGraphProcessor` by `openupm add com.alelievr.node-graph-processor`
3. Open unity project
4. Expect `Assets/Create` have an option to create Graph, but there's none.

`Packages/manifest.json`
``` json
{
  "dependencies": {
    "com.unity.ide.rider": "2.0.7",
    "com.unity.ugui": "1.0.0",
    "com.unity.modules.ai": "1.0.0",
    "com.unity.modules.androidjni": "1.0.0",
    "com.unity.modules.animation": "1.0.0",
    "com.unity.modules.assetbundle": "1.0.0",
    "com.unity.modules.audio": "1.0.0",
    "com.unity.modules.cloth": "1.0.0",
    "com.unity.modules.director": "1.0.0",
    "com.unity.modules.imageconversion": "1.0.0",
    "com.unity.modules.imgui": "1.0.0",
    "com.unity.modules.jsonserialize": "1.0.0",
    "com.unity.modules.particlesystem": "1.0.0",
    "com.unity.modules.physics": "1.0.0",
    "com.unity.modules.physics2d": "1.0.0",
    "com.unity.modules.screencapture": "1.0.0",
    "com.unity.modules.terrain": "1.0.0",
    "com.unity.modules.terrainphysics": "1.0.0",
    "com.unity.modules.tilemap": "1.0.0",
    "com.unity.modules.ui": "1.0.0",
    "com.unity.modules.uielements": "1.0.0",
    "com.unity.modules.umbra": "1.0.0",
    "com.unity.modules.unityanalytics": "1.0.0",
    "com.unity.modules.unitywebrequest": "1.0.0",
    "com.unity.modules.unitywebrequestassetbundle": "1.0.0",
    "com.unity.modules.unitywebrequestaudio": "1.0.0",
    "com.unity.modules.unitywebrequesttexture": "1.0.0",
    "com.unity.modules.unitywebrequestwww": "1.0.0",
    "com.unity.modules.vehicles": "1.0.0",
    "com.unity.modules.video": "1.0.0",
    "com.unity.modules.vr": "1.0.0",
    "com.unity.modules.wind": "1.0.0",
    "com.unity.modules.xr": "1.0.0",
    "com.alelievr.node-graph-processor": "1.0.0"
  },
  "scopedRegistries": [
    {
      "name": "package.openupm.com",
      "url": "https://package.openupm.com",
      "scopes": [
        "com.openupm",
        "com.alelievr.node-graph-processor"
      ]
    }
  ]
}
```

**Comment by alelievr** [comment]
Hello, When NGP is installed as a package, it doesn't include the context menu actions to avoid polluting the menus with stuff that you may not need (especially because NGP is intended to be used as a dependency for a specific graph project).

That's why the context menu options are contained in a sample of the package. That being said it's apparently broken in the current version, so I'll try to solve this issue (I leave this issue open until I fix it).
![image](https://user-images.githubusercontent.com/6877923/107875945-429a0400-6ec3-11eb-9c71-7fcc3c54163d.png)


Thanks for the bug report!

**Comment by alelievr** [comment]
This has been fixed in the 1.1.0 version
